content/en/docs/apidocs-mxsdk/apidocs/design-properties.md

```diff
@@ -13,25 +13,38 @@ The Mendix Design Properties API allows you to create or extend design propertie
 
 To use the API, you need to alter the *design-properties.json* file of a specific module in your application's **themesource** folder. This process is described in the [Design Properties Definitions](#design-properties-definitions) section below.
 
-This guide outlines how design properties work and can help you create custom design properties. Many apps can simply use the Atlas UI theme and its included set of design properties to satisfy their styling needs. Atlas UI provides design properties that are built on top of the functionality described in this guide. So while design properties from Atlas UI are used as examples below, design properties themselves are not only for the Atlas UI theme. In fact, if you want to customize your styling more deeply, you will have to create your own custom design properties.
+Many apps can simply use the Atlas UI theme and its included set of design properties to statisfy their styling needs. However, if you want to customize your styling more deeply, you will have to create your own custom design properties. This guide outlines how design properties work and can help you create custom design properties. The design properties provided by Atlas UI have been build in the same way as outlined here.
 
 Design properties are a special set of settings shipped together with a Mendix theme module. Design properties are shared among all the Mendix apps which use a specific theme module. 
 
-In Studio Pro, you can see which design properties are available for a widget either in the **Properties** pane or in the widget's settings dialog box under the **Appearance** tab. 
+In Studio Pro, you can see which design properties are available for a widget in the **Properties** pane. A limited set of design properties can also be found in the widget's settings dialog box under the **Appearance** tab.
 
 ## 2 Using Design Properties
 
 While styling Mendix apps, users must often apply the same set of CSS or native styling classes to widgets on different pages again and again. This work is time consuming and vulnerable to human error as you edit text fields in order to apply classes to a widget.
 
 Design properties can make this work easier and safer. By configuring your own custom design properties, a certain styling can be applied to a widget in a few clicks.
 
+You can define the styles that correspond to your design properties as CSS classes. Starting with Mendix 10, you can also use design properties to apply CSS variables (also known as CSS custom properties) to specified CSS properties. In this document, we will show examples of both ways.
```

Comment:
This looks like it could be a good place for a sub heading or some other way of flowing into this description, it doesn't flow intuitively from "Using Design Properties"